package.json

```diff
@@ -12,7 +12,8 @@
     "format": "prettier --write \"**/*.{ts,tsx}\"",
     "lint": "turbo lint --",
     "prepare": "husky install",
-    "test": "turbo test"
+    "test": "turbo test",
+    "add-rn-version": "./scripts/add-rn-version"
```

Comment:
this file contains devDependencies to some RN config(babel, metro). Can we also update the versions to 0.81 there?

Reply:
Which file are you referring to? there isn't any dev deps in the root.